README.md

```diff
@@ -179,6 +179,14 @@ and gene expression prediction on the _Pseudomonas aeruginosa_ dataset. The chec
 ## Citations <a name="citations"></a>
 If you find GeneBac useful in your work, please cite our paper:
 ```bibtex
+@article{wiatrak2024genebac,
+  author = {Wiatrak, Maciej and Weimann, Aaron and Dinan, Adam M and Brbić, Maria and Floto, R. Andres},
+  title={Sequence-based modelling of bacterial genomes enables accurate antibiotic resistance prediction},
+  year={2024},
+  doi={10.1101/2020.12.15.422761},
+  url={https://www.biorxiv.org/content/10.1101/2020.12.15.422761v1},
```

Comment:
URL seems to point ot a different paper called "Transformer protein language models are unsupervised structure learners"